CHANGELOG.md

```diff
@@ -6,15 +6,17 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased] - XXXX-XX-XX
 
-
-## [1.5.1] - 2021-05-28
```

Comment:
There is a 1.5.1 tag https://github.com/owncloud/encryption/releases/tag/v1.5.1

Why is this heading being deleted?